cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -160,21 +160,22 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
             """This is the same command as `SHOW ALL ROLES`.
               |When first starting a Neo4j DBMS there are a number of built-in roles:
               |
-              |* `reader` - can perform read-only queries on all databases except `system`
-              |* `editor` - can perform read and write operations on all databases except `system`, but cannot make new labels or relationship types
+              |* `reader` - can perform read-only queries on all databases except `system`.
+              |* `editor` - can perform read and write operations on all databases except `system`, but cannot make new labels or relationship types.
               |* `publisher` - can do the same as `editor`, but also create new labels and relationship types.
-              |* `architect` - can do the same as `architect` as well as create and manage indexes and constraints
-              |* `admin` - can do the same as all the above, as well as manage databases, users, roles and privileges
+              |* `architect` - can do the same as `publisher` as well as create and manage indexes and constraints.
+              |* `admin` - can do the same as all the above, as well as manage databases, users, roles and privileges.
               |""".stripMargin)
           resultTable()
         }
         p("There are multiple versions of this command, the default being `SHOW ALL ROLES`. " +
           "To only show roles that are assigned to users, the command is `SHOW POPULATED ROLES`. " +
           "To see which users are assigned to roles `WITH USERS` can be appended to the commands. " +
-          "This will give one result row for each user, so if a role is assigned to two users then it will show up twice in the result.")
+          "This will give one result row for each user, so if a role is assigned to two users then it will show up twice in the result. ")
         query("SHOW POPULATED ROLES WITH USERS", assertRolesShown(Seq("admin"), Seq("PUBLIC"))) {
-          p("The table of results contains two columns, the first is the role name, and the other a flag indicating whether the role is a built-in role or a custom role.")
+        p("The table of results will show information about the role, i.e. its name, whether or not it is built-in and what database it belongs to. ")
```

Comment:
I think it might be a good idea not to reference `isBuiltIn` at all, due to the Cypher team wanting to remove this column.

Perhaps something like:
> The table of results will show information about the role and what database it belongs to.

Reply:
Agree!